.travis.yml

```diff
@@ -1,25 +1,27 @@
 sudo: true
 
-language: android
+language: java
 jdk: oraclejdk8
 
 dist: precise
 
-android:
-  components:
-    - tools
-    - platform-tools
-    - tools
-    - build-tools-26.0.0
-    - android-26
-    - extra-android-m2repository
-
 before_script:
   - export SKIP_OKBUCK=true
   - export EXTRA_OKBUCK_ARGS="--quiet --stacktrace"
-  - mkdir -p $ANDROID_HOME/licenses
-  - echo $android_sdk_license > $ANDROID_HOME/licenses/android-sdk-license
-  - echo $android_sdk_preview_license > $ANDROID_HOME/licenses/android-sdk-preview-license
+  - export ANDROID_HOME="$HOME"/android-sdk
+  - mkdir -p "$ANDROID_HOME"
+  - export ANDROID_SDK_FILE_NAME=sdk-tools-darwin-3859397.zip
```

Comment:
why darwin? travis runs on linux right?

Reply:
ah, facepalm, now I'm curious how it works in other projects where I did same thing… I guess Android tooling is more or less crossplatform now, that's how you check that lol